specification/k3po.control/src/main/scripts/org/kaazing/specification/control/accept.error.if.no.prepare.rpt

```diff
@@ -0,0 +1,68 @@
+#
+# Copyright 2007-2015, Kaazing Corporation. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+accept tcp://localhost:12345
+accepted
+connected
+
+read "START\n"
+read "\n"
+
+write "ERROR\n"
```

Comment:
This set of errors is in one test just for simplicity. Normally they will not occur like this. Anyway, the driver responds based on the state and I see no problem here.

If we are suppose to close connection on the driver, then we will have to stop the execution of the script. Also, if there is any result to be transmitted to the client it should not be done (a FINISHED is sent even if an ABORT occurs). 
From my point of view, this will complicate a lot the flow on the driver. Currently, it is the responsibility of the client to restart the connection if an error occurs, and the client should decide if he wants to restart or continue from this state. 

What you propose makes sense from a Junit test's point of view, because any kind of error will fail the test. But in case of a text connection to the driver, for example, or other type of usage, it might not be that useful. If I mistake the name of a barrier I need to restart from scratch instead of just trying again with the correct name.